src/plugins/discover/public/components/discover_grid/discover_grid.tsx

```diff
@@ -182,7 +189,7 @@ export const DiscoverGrid = ({
   useNewFieldsApi,
   isSortEnabled = true,
   isPaginationEnabled = true,
-  controlColumnIds = ['openDetails', 'select'],
+  controlColumnIds = CONTROL_COLUMN_IDS_DEFAULT,
```

Comment:
:information_source: Since we're actually calling this component without specifying `controlColumnIds`, we're using the default value. But inlining that default value causes this array to be a new one every time we render, thus memoization not to work.